osu.Game/Overlays/VolumeOverlay.cs

```diff
@@ -36,10 +36,12 @@ public class VolumeOverlay : VisibilityContainer
 
         public Bindable<bool> IsMuted { get; } = new Bindable<bool>();
 
+        protected readonly IBindable<OverlayActivation> OverlayActivationMode = new Bindable<OverlayActivation>(OverlayActivation.All);
+
         private SelectionCycleFillFlowContainer<VolumeMeter> volumeMeters;
 
         [BackgroundDependencyLoader]
-        private void load(AudioManager audio, OsuColour colours)
+        private void load(AudioManager audio, OsuColour colours, OsuGame osuGame)
```

Comment:
This is going to break tests that don't provide an `OsuGame`. You'll want to allow nullable dependencies by changing the BDL attribute above to `[BackgroundDependencyLoader(true)]`.